server/lib/notifications.js

```diff
@@ -106,6 +106,7 @@ async function notifySubscribers(users, activity, options = {}) {
       if (data.host) {
         data.actions = {
           approve: `${config.host.website}/${data.host.slug}/collectives/${data.collective.id}/approve`,
+          reject: `${config.host.website}/${data.host.slug}/collectives/${data.collective.id}/reject`,
```

Comment:
The link doesn't work

Reply:
one option could be redirecting to `reject: `${config.host.website}/${data.host.slug}/dashboard/pending-applications`,`